sw/host/opentitanlib/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+#[macro_use]
+extern crate lazy_static;
+#[macro_use]
+extern crate bitflags;
+#[macro_use]
```

Comment:
I don't think you need `#[macro_use]` here because you're using Rust 2018: https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html